docs/orchestrating-elastic-stack-applications/elasticsearch/storage-recommendations.asciidoc

```diff
@@ -0,0 +1,66 @@
+:parent_page_id: elasticsearch-specification
+:page_id: storage-recommendations
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Storage recommendations
+
+ECK does not come with its own storage mechanism for Elasticsearch data. It is compatible with any Kubernetes storage option. It is recommended to use PersistentVolumes, by configuring the <<{p}-volume-claim-templates,VolumeClaimTemplates>> section of the Elasticsearch resource.
+
+Multiple PersistentVolume storage classes are available, depending on your Kubernetes setup. Their specifications impact Elasticsearch performance and operations. There is no best solution to suit everyone: it is important to understand the trade-offs between various available options.
+
+[float]
+== Network-attached or Local PersistentVolumes
+
+PersistentVolumes come in two flavours, with a different performance/price/operations trade-off. ECK handles both the same way.
+
+**Network-attached PersistentVolumes** can generally be attached to a Pod regardless of the host they are scheduled on.
```

Comment:
It's not exactly true for all providers since some network-attached volumes may be bound to a single host.
Is it ok to keep "generally" to nuance that statement?